### PR TITLE
Hide irrelevant Matrix Id 

### PR DESCRIPTION
**Pull request by ClaireGizard**
https://github.com/tchapgouv/tchap-android-v2/issues/272
https://github.com/tchapgouv/tchap-android-v2/issues/273

Room members list:
Screenshot_1638377739

Room member details:
Screenshot_1638439668

### CODE REVIEW

File: vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt [hunk 1]
```diff
@@ -43,13 +44,15 @@ abstract class BaseProfileMatrixItem<T : ProfileMatrixItem.Holder> : VectorEpoxy
...
-                // Special case for ThreePid fake matrix item
-                .takeIf { it != "@" }
+//        val matrixId = matrixItem.id
+//                .takeIf { it != bestName }
+//                // Special case for ThreePid fake matrix item
+//                .takeIf { it != "@" }
         holder.view.onClick(clickListener?.takeIf { editable })
         holder.titleView.text = bestName
-        holder.subtitleView.setTextOrHide(matrixId)
+        //holder.subtitleView.setTextOrHide(matrixId)
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt:53]: Please comment this line as above (`//` at the beginning of the line)

File: vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt [hunk 2]
```diff
@@ -43,13 +44,15 @@ abstract class BaseProfileMatrixItem<T : ProfileMatrixItem.Holder> : VectorEpoxy
...
+        // Hide the Matrix Id for Tchap
```
Comment (yostyle) [inline vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt:54]:
```suggestion
        // Tchap: Hide the Matrix Id
```

Comment (Florian14) [inline vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt:54]: Please move this comment above the commented line

File: vector/src/main/res/layout/view_stub_room_member_profile_header.xml [hunk 3]
```diff
@@ -116,4 +117,4 @@
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-</im.vector.app.core.platform.StateView>
\ No newline at end of file
+</im.vector.app.core.platform.StateView>
```
Comment (yostyle) [inline vector/src/main/res/layout/view_stub_room_member_profile_header.xml:120]: You should undo this change. Please keep the code coming from Element.

File: vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt [hunk 4]
```diff
@@ -43,13 +44,15 @@ abstract class BaseProfileMatrixItem<T : ProfileMatrixItem.Holder> : VectorEpoxy
...
+        holder.subtitleView.visibility = View.GONE
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/core/epoxy/profiles/BaseProfileMatrixItem.kt:55]: Please use androidx `isVisible` extension instead of changing the visibility manually

File: TCHAP_CHANGES.md [hunk 5]
```diff
@@ -3,6 +3,9 @@ Changes in Tchap 1.99.3 (2021-xx-xx)
 
 Bugfix 🐛:
  - [Login Screen] An unexpected error on the password field #276
+
+Improvements 🙌:
+ - Remove irrelevant Matrix Id PR #294
```
Comment (Florian14) [inline TCHAP_CHANGES.md:8]:
```suggestion
 - Remove irrelevant Matrix Id #294
```

**Review by Florian14 — COMMENTED** [review]
SGTM. Please, could you anonymize screenshots in your PR?

File: TCHAP_CHANGES.md (hunk 5 above)
Comment (ClaireGizard) [reply TCHAP_CHANGES.md:8]: @giomfo ask me to put the PR to make a reference to the PR and not to the ISSUE.

Comment (Florian14) [reply TCHAP_CHANGES.md:8]: ACK

**Review by giomfo — APPROVED** [review]
LGTM
